test/dashjs.test.js

```diff
@@ -17,7 +17,12 @@
       src: 'movie.mpd',
       type: 'application/dash+xml'
     },
-    testHandleSource = function(assert, source, expectedKeySystemOptions, limitBitrateByPortal) {
+
+    testHandleSource = function(assert, source, expectedKeySystemOptions, testData) {
+      if (testData === undefined) {
+        testData = {};
+      }
+      var eventHandlers = (testData.eventHandlers !== undefined) ? testData.eventHandlers : {};
```

Comment:
I'm not sure if the linter is set up, but usually we use `typeof testData.eventHandlers !== 'undefined'`

Reply:
Though if you expect to always use an object and not pass in an array it should be able to just test the truthiness

Reply:
In the process of doing this PR, I noticed that it's not linting the `test` directory. I was gonna just tack it on to this PR, but it required a few line breaks here and there so opted not to. I'll open another PR with the changes.

> Though if you expect to always use an object and not pass in an array it should be able to just test the truthiness
Thanks, I'd actually meant to change it to just a truthiness check but I guess I got distracted.